machine_configuration/mco-coreos-layering.adoc

```diff
@@ -12,7 +12,7 @@ toc::[]
 [id="coreos-layering-about_{context}"]
 == About {op-system} image layering
 
-Image layering allows you to customize the underlying node operating system on any of your cluster worker nodes. This helps keep everything up-to-date, including the node operating system and any added customizations such as specialized software.
+Image layering allows you to customize the underlying node operating system on any of your cluster nodes. This helps keep everything up-to-date, including the node operating system and any added customizations such as specialized software.
```

Comment:
Image layering is allowed on all nodes, regardless of what MachineConfigPool they belong to. This change clarifies that.